x-pack/apm-server/sampling/processor.go

```diff
@@ -160,27 +228,29 @@ func (p *Processor) processTransaction(tx *model.Transaction) (bool, error) {
 		// This is a local optimisation only. To avoid creating network
 		// traffic and load on Elasticsearch for uninteresting root
 		// transactions, we do not propagate this to other APM Servers.
-		return true, p.storage.WriteTraceSampled(tx.TraceID, false)
+		return true, false, p.storage.WriteTraceSampled(tx.TraceID, false)
 	}
 
 	// The root transaction was admitted to the sampling reservoir, so we
 	// can proceed to write the transaction to storage and then drop it;
 	// we may index it later, after finalising the sampling decision.
-	return true, p.storage.WriteTransaction(tx)
+	return true, true, p.storage.WriteTransaction(tx)
```

Comment:
Actually reading this comment - what is the meaning of `dropped` in this context?